lib/ex_doc/formatter/html/templates.ex

```diff
@@ -120,6 +120,11 @@ defmodule ExDoc.Formatter.HTML.Templates do
     |> h()
   end
 
+  @doc false
```

Comment:
It works fine when private and I don't think a similar issue will arise with `epub`s, I'll make it private.